modules/functional-test/src/main/java/com/intuit/wasabi/tests/service/assignment/CreateEventForUnassignedUserTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * ******************************************************************************
+ *  * Copyright 2016 Intuit
```

Comment:
@sudheerkethireddy please use this class as an example of how the license file should look like: https://github.com/intuit/wasabi/blob/develop/modules/functional-test/src/main/java/com/intuit/wasabi/tests/service/assignment/BasicAssignment.java